community/knative/charts/serving/charts/monitoring/charts/prometheus/templates/prometheus-default-system-role.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Values.prometheusSystem.rb.roleName | quote }}
```

Comment:
For the following files, the metadata.name on their Roles is "prometheus-system" and the metadata.name on their RoleBindings is also prometheus-system:
- charts/serving/charts/monitoring/charts/prometheus/templates/prometheus-default-system-role.yaml
- charts/serving/charts/monitoring/charts/prometheus/templates/prometheus-istio-system-role.yaml
- charts/serving/charts/monitoring/charts/prometheus/templates/prometheus-knative-monitoring-system-role.yaml
- charts/serving/charts/monitoring/charts/prometheus/templates/prometheus-kube-system-role.yaml

Reply:
Will be fixed in next commit